prisma/migrations/20250725002458_user/migration.sql

```diff
@@ -0,0 +1,13 @@
+-- CreateTable
+CREATE TABLE "User" (
+    "id" TEXT NOT NULL,
+    "name" TEXT NOT NULL,
+    "email" TEXT NOT NULL,
+    "password" TEXT NOT NULL,
+    "birthDate" TIMESTAMP(3) NOT NULL,
+
```

Comment:
```suggestion
"createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
"updatedAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP
```

É uma boa prática adicionar campos de auditoria nas tabelas criadas